content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -0,0 +1,265 @@
+
+The **authorization.k8s.io/v1beta1** API version of LocalSubjectAccessReview, SelfSubjectAccessReview, and SubjectAccessReview will no longer be served in v1.22.
+
+* Migrate manifests and API clients to use the **authorization.k8s.io/v1** API version, available since v1.6.
+* Notable changes:
+    * `spec.group` was renamed to `spec.groups` in v1 (fixes [#32709](https://github.com/kubernetes/kubernetes/issues/32709))
+
+#### CertificateSigningRequest
+
+The **certificates.k8s.io/v1beta1** API version of CertificateSigningRequest will no longer be served in v1.22.
+
+* Migrate manifests and API clients to use the **certificates.k8s.io/v1** API version, available since v1.19.
+* All existing persisted objects are accessible via the new API
+* Notable changes in `certificates.k8s.io/v1`:
+    * For API clients requesting certificates:
+        * `spec.signerName` is now required (see [known Kubernetes signers](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#kubernetes-signers)), and requests for `kubernetes.io/legacy-unknown` are not allowed to be created via the `certificates.k8s.io/v1` API
+        * `spec.usages` is now required, may not contain duplicate values, and must only contain known usages
+    * For API clients approving or signing certificates:
+        * `status.conditions` may not contain duplicate types
+        * `status.conditions[*].status` is now required
+        * `status.certificate` must be PEM-encoded, and contain only `CERTIFICATE` blocks
+
+#### Lease
+
+The **coordination.k8s.io/v1beta1** API version of Lease will no longer be served in v1.22.
+
+* Migrate manifests and API clients to use the **coordination.k8s.io/v1** API version, available since v1.14.
+* All existing persisted objects are accessible via the new API
+* No notable changes
+
+#### Ingress
+
+The **extensions/v1beta1** and **networking.k8s.io/v1beta1** API versions of Ingress will no longer be served in v1.22.
+
+* Migrate manifests and API clients to use the **networking.k8s.io/v1** API version, available since v1.19.
+* All existing persisted objects are accessible via the new API
+* Notable changes:
+    * `spec.backend` is renamed to `spec.defaultBackend`
+    * The backend `serviceName` field is renamed to `service.name`
+    * Numeric backend `servicePort` fields are renamed to `service.port.number`
+    * String backend `servicePort` fields are renamed to `service.port.name`
+    * `pathType` is now required for each specified path. Options are `Prefix`, `Exact`, and `ImplementationSpecific`. To match the undefined `v1beta1` behavior, use `ImplementationSpecific`.
+
+#### IngressClass
+
+The **networking.k8s.io/v1beta1** API version of IngressClass will no longer be served in v1.22.
+
+* Migrate manifests and API clients to use the **networking.k8s.io/v1** API version, available since v1.19.
+* All existing persisted objects are accessible via the new API
+* No notable changes
+
+#### RBAC
```

Comment:
To make it clear that this isn't talking about a RBAC resource where you could `kubectl get rbac`, I recommend grouping them as:
```suggestion
#### RBAC resources
```